discovery-infra/test_infra/controllers/node_controllers/host.py

```diff
@@ -63,6 +63,17 @@ def kill_service(self, service):
         logging.info("Killing service %s on host %s", service, self.name)
         self.run_command(f'sudo systemctl kill {service}.service || true')
 
+    def kill_installer(self):
+        current_proc = self.run_command("sudo su root -c 'podman ps | grep assisted-installer'")
+        logging.info(f"Current running installer container details on {self.name}: {current_proc}")
+        container_id = self.run_command("sudo su root -c '(podman ps | grep assisted-installer | cut -d \" \" -f1)'")
```

Comment:
why u need to run twice the command? u have string already split it ?

Reply:
i think that easier to have function that tell , kill_podman_container and provide container name to it.
another way if i will want to kill next_step container i will need to write same get id code

Reply:
my thought was that its better to delete container by id. In any way kill_podman_container_by_id will work also with container name. it is podman kill command. Do you want me to change this kill_podman_container_by_id method name?

Reply:
yeah sounds better for me.

Reply:
it can be by id but then u need a function that gets id from name